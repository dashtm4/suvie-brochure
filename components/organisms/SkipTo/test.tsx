import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { SkipTo } from '.'

describe('<SkipTo />', () => {
  it('should renders correct <SkipTo />', () => {
    const wrapper = shallow(<SkipTo />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
