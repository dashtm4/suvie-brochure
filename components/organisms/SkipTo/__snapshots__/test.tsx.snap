// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SkipTo /> should renders correct <SkipTo /> 1`] = `
<div
  className="fixed top-0 left-1/2 z-50 p-3 bg-white rounded transform -translate-x-1/2 -translate-y-full max-w-sm w-full focus-within:shadow-2xl focus-within:translate-y-0"
>
  <div
    className="mb-2 font-bold"
  >
    Skip to
  </div>
  <ul />
</div>
`;
